src/app/zap-templates/zcl/data-model/chip/device-energy-management-cluster.xml

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="Energy Management"/>
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0098"/>
+    <field name="PowerAdjustment" mask="0x1"/>
+    <field name="PowerForecastReporting" mask="0x2"/>
+    <field name="StateForecastReporting" mask="0x4"/>
+    <field name="ForecastAdjustment" mask="0x8"/>
+  </bitmap>
+  <cluster apiMaturity="provisional">
+    <name>Device Energy Management</name>
+    <domain>Energy Management</domain>
+    <code>0x0098</code>
+    <define>DEVICE_ENERGY_MANAGEMENT_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
```

Comment:
Added Revision 2 (since 1 was the initial, 2 is the fixes after 0.7 ballot comments).
    
    <globalAttribute side="server" code="0xFFFD" value="2" />